txpipe/plotting/correlations.py

```diff
@@ -339,13 +456,20 @@ def make_theory_plot_data(data, cosmo, obs, label, smooth=True, xi=None):
         for j in range(i+1):
             print(f"Computing theory lensing-lensing ({i},{j})")
 
+            # uncomment to take ratio in Fourier space to get the exact ells
```

Comment:
Is the i==0 and j==0 test here just to do it once?  Any reason not to do it each time in case ell changes?